index.js

```diff
@@ -122,8 +123,23 @@ module.exports = (input, opts) => {
 		},
 		{
 			title: 'Bumping version using npm',
-			enabled: () => opts.yarn === false,
+			enabled: () => opts.yarn === false && !opts.contents,
 			task: () => exec('npm', ['version', input])
+		},
+		{
+			title: 'Bumping version using npm',
```

Comment:
You can perfectly merge this task together with the previous task. You can always pass in `cwd` and use `opts.contents || ''` or something.

Reply:
Good point! Will refactor! 👍